src/zenml/integrations/mlflow/__init__.py

```diff
@@ -31,15 +31,14 @@ class MlflowIntegration(Integration):
     """Definition of MLflow integration for ZenML."""
 
     NAME = MLFLOW
-    # We need to pin protobuf to a version <=4 here, as this mlflow release
-    # does not pin it. They fixed this in a later version, so we can probably
-    # remove this once we update the mlflow version.
+
     REQUIREMENTS = [
         "mlflow>=2.1.1,<=2.12.2",
         "mlserver>=1.3.3",
         "mlserver-mlflow>=1.3.3",
         # TODO: remove this requirement once rapidjson is fixed
         "python-rapidjson<1.15",
+        "pydantic>=2.7.0,<2.8.0"
```

Comment:
Can you add a comment, why we pin it here. Also pinning here is not aligned with `pyproject.toml` (by gist, it is, but syntactically different)